Symfony/Sniffs/Commenting/AnnotationsSniff.php

```diff
@@ -28,13 +28,7 @@
  */
 class AnnotationsSniff implements Sniff
 {
-
-    /**
-     * Pattern
-     *
-     * @var string
-     */
-    private static $_pattern = '/^@([^\\\(]+).*$/i';
+    const PATTERN = '/^@([^\\\(]+).*$/i';
```

Comment:
Probably a PHPUnit/xdebug problem that this is not claimed covered in the `preg_replace` calls, but it's IMHO cleaner as a constant anyway...